sympy/core/relational.py

```diff
@@ -203,6 +203,44 @@ def __nonzero__(self):
 
     __bool__ = __nonzero__
 
+    def as_solution_set(self, gen=False):
+        """
+        Rewrites the simplified (solution) inequality in terms of real set
+
+        Examples
+        ========
+
+        >>> from sympy import Symbol, Eq
+        >>> x = Symbol('x', real=True)
+        >>> (x>0).as_solution_set()
+        (0, oo)
+        >>> Eq(x, 0).as_solution_set()
+        {0}
+        >>> Ge(y, x).as_solution_set(y)
+        [x, oo)
+        >>> Lt(y, x).as_solution_set(y)
+        (-oo, x)
+        """
+        from sympy.solvers.inequalities import solve_univariate_inequality
+        syms = self.free_symbols
+
+        if len(syms) == 1:
+            sym = syms.pop()
+            return solve_univariate_inequality(self, sym, relational=False)
+        else:
+            from sympy import Interval
+            e = self.lhs - self.rhs
```

Comment:
If this function only makes sense when the input is a Relational, then it should be a method of Relational.